models/dataset.go

```diff
@@ -112,6 +112,7 @@ type Dataset struct {
 	IsBasedOn         *IsBasedOn       `bson:"is_based_on,omitempty"            json:"is_based_on,omitempty"`
 	CanonicalTopic    *Topic           `bson:"canonical_topic,omitempty"        json:"canonical_topic,omitempty"`
 	SubTopics         []Topic          `bson:"sub_topics,omitempty"             json:"sub_topics,omitempty"`
+	Survey            *string          `bson:"survey,omitempty"                 json:"survey,omitempty"`
```

Comment:
Is the pointer required here? I don't think we ever send a null value and it wouldn't be consistent with the rest of the struct